### ISSUE TITLE
"Files are newer on disk" when opening project twice for multiplayer

### ISSUE THREAD

**Issue by ottovw** [godotengine/godot#66473]
### Godot version

4.0-beta1

### System information

Windows 10, Vulkan,  Nvidia 2080

### Issue description

When opening the same project twice, e.g. for testing multiplayer and all changes are saved a popup with "Files are newer on disk" occurrs regularly.  If you choose reload or save it goes away and reappears several moments later.

### Steps to reproduce

* Have a godot project in editor 1) open
* Save all files (no pending changes) (optional step?)
* Have a scene running in editor 1)  (optional step?)
* Open same project in second editor 2)
* Tab over to editor 1)  and the popup appears shortly
* Choose any action (Save, Cancel, Do nothing, ...)
* Tab over to editor 2)
* Same thing happens in the other editor 2)

**Comment by Calinou** [comment]
This is expected. Opening the same project more than once in the editor is not supported.

**Comment by ottovw** [comment]
@Calinou thanks for clarification. Is there documentation available?

- When running two instances how can i set different flags for instances (one being server, the other client). i currently use a flag on a node

**Comment by ottovw** [comment]
@Calinou thanks for clarification. Is there documentation available?

Some notes:
- First I couldn't find the two instances, because they appear overlapping 100%. Found out by chance, that's a bit unintuitive.
- When running two instances how can i set different flags for instances (one being server, the other client). i currently use a flag on a node.
- Would be nice to have a warning "Project already opened, which is not supported" or the like

**Comment by Calinou** [comment]
> Is there documentation available?

No, but we should warn when opening more than once instance on the same project. This can likely be achieved by creating a lockfile in the system's temporary folder (not the project's `.godot/` folder), and removing this file when the editor quits. Writing this file to a temporary folder will ensure the file is never sent to other users when zipping the project files, and will prevent the file from lingering forever in case of a crash (as the temporary folder is periodically cleaned by the OS).

> When running two instances how can i set different flags for instances (one being server, the other client). i currently use a flag on a node.

See https://github.com/godotengine/godot-proposals/issues/522.
